one_shot_scripts/fix_followers.js

```diff
@@ -3,6 +3,8 @@ require('dotenv').config();
 
 const mongoose = require('mongoose');
 
+// the below import does not exist and should be updated if we ever wish to run this script again
```

Comment:
@dekkerglen Should this script just be deleted at this point?

Reply:
it was just a maintenance script so we don't need it anymore